src/libs/OptionsListUtils.ts

```diff
@@ -359,6 +359,12 @@ Onyx.connect({
     callback: (value) => (allReportsDraft = value),
 });
 
+let activePolicyID: OnyxEntry<string>;
```

Comment:
Good point from @allgandalf, let's keep it activePolicyID though as thats how we call it everywhere else and actually rename it in the IOU.ts file to activePolicyID too please @nkdengineer https://github.com/Expensify/App/blob/55f67f53ca8ed8596943ce930ec10a3bd6f13427/src/libs/actions/IOU.ts#L272-L276

this was it will be cleaner

Reply:
Thanks, I updated.

Reply:
@mountiny I updated. Sorry for the confusion.